reports/transmissibility.Rmd

```diff
@@ -559,6 +559,47 @@ last_trends %>%
 
 ## Results
 
+```{r}
+library(R0)
+
+mGT <- generation.time("gamma", c(3, 1.5))
```

Comment:
We can standardize this at a later stage. I like working with `distrcrete` objects, but it currently requiers ad-hoc code: 
https://github.com/epiverse-trace/data_pipelines/blob/main/reports/transmissibility.Rmd#L452-L461

To clarify: 
* generation time = time interval between dates of infection of infector -> infectee
* serial interval = time interval between dates of onset of infector -> infectee
They are assumed to have the same mean but different variances. In practice, when we need the GT, we most often use the SI as a proxy, as the GT is much harder to get to (dates of onset are observed, dates of infection, more rarely).